src/main/java/easy/gc_coffee_api/dto/UpdateMenuRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package easy.gc_coffee_api.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class UpdateMenuRequestDto {
+    private String name;
+    private int price;
+    private String category;
+    private Long imageId;
```

Comment:
잘 작성해주셨는데 검증로직을 이런식으로 작성해주시면 좋을 것 같아요
스프링 validation 의존성 있어서 굳이 유스케이스에서 작성 안하셔도 될 것 같아요!
```suggestion
    @NotBlank(message = "메시지~")
    private String name;
    @NotNull
    @Min(0)
    private int price;
    @NotBlank(message = "메시지~")
    private String category;
    private Long imageId;
```

Reply:
컨트롤러 계층에서 유효성검증을 진행하기 위한 목적이 맞을까요??

Reply:
넵넵 맞습니다!

Reply:
저는 지금 태호님이 하신 방식을 더 좋아하긴 하는데 validation 쓰면 확실히 편하긴 합니다. 참고만 해주세용

Reply:
둘 다 장점이 있는 것 같네요! validation이 간결하고 편하긴한데 usecase에서의 명시적인것도 중요하다고 생각해서 둘 다 구현해둔 상태로 두었습니다